js/inspect/inspect.js

```diff
@@ -92,7 +92,12 @@ const drop = (document) => {
   // XSS review - no problem here
   div.innerHTML = '<h1>H1</h1><h2>H2</h2><h3></h3><h4></h4><h5></h5><h6></h6><a href="#">Link</a><p>Some text</p>';
 
-  center.parentElement.insertBefore(div, center);
+  if (center.parentElement) {
```

Comment:
```suggestion
  if (center && center.parentElement) {
```

Just to cover the case: no center found and no h2 found.